`<IconButton/>`- Adding tiny and large styling & updating documentation

TalZa1810 opened the pull request.

shlomitc commented on packages/wix-ui-core/src/themes/backoffice/icon-button/icon-button.st.css:
```diff
@@ -193,19 +193,30 @@
 }
 
+.iconButton.large {
+  width: 49px;
```
why 49?

TalZa1810 replied:
According to Zeplin:
![lll](https://user-images.githubusercontent.com/24567980/58813256-0873ca80-862c-11e9-8ef3-2269b31e8e61.png)

shlomitc replied:
doesn't seem right - @LittleTal ?

LittleTal replied:
@shlomitc @TalZa1810 It's a mistake, should be 48

TalZa1810 replied:
@shlomitc @LittleTal 
Fixed

TalZa1810 commented:
Hey @mosheKerbel ,
I need the styles in wix-ui-core in order to resolve this issue:
https://github.com/wix/wix-style-react/issues/3431 

I've implemented both tiny & large and added the examples under the title:
`Sizes - Icon Buttons (sizes)`:
https://wix-wix-ui-core-pr-1326.surge.sh/?path=/story/backoffice--buttons.


Thanks :)

mosheKerbel commented:
@TalZa1810 looks like tests are flaky, @jonathanadler suggested to create some dummy commit in order to re-trigger the build.
